util/class.tx_mkforms_util_FormBase.php

```diff
@@ -559,7 +559,8 @@ public static function addFields(array $data, array $fields)
                     unset($data[$fieldKey]);
                 } // Den Wert setzen oder Überschreiben
                 elseif (!$bUnset) {
-                    $bOverride = isset($fields[$fieldKey.'.']) && isset($fields[$fieldKey.'.']['override']) && (intval($fields[$fieldKey.'.']['override']) > 0);
+                    $bOverride = isset($fields[$fieldKey.'.']) && isset($fields[$fieldKey.'.']['override']) && ((int)$fields[$fieldKey
+                            . '.']['override'] > 0);
```

Comment:
Please remove the new line break or add line breaks after each part of the condition.

Reply:
Done.